src/notation/internal/notationinteraction.h

```diff
@@ -248,6 +248,7 @@ class NotationInteraction : public INotationInteraction, public async::Asyncable
         Ms::EditData ed;
         std::vector<Element*> elements;
         std::vector<std::unique_ptr<Ms::ElementGroup> > dragGroups;
+        DragMode mode { DragMode::BothXY };
```

Comment:
We usually use initialization with `=` everywhere. It seems to be easier to read :) But it doesn't matter, initialization with `{}` is more universal.